### DISCUSSION TITLE
v1.5.15-00

### DISCUSSION THREAD

**Discussion by gaborauth** [gaborauth/toolsboox-android#162] [Announcements]
- Added experimental cloud backup ability
- Code quality improvements
- Small bugfixes

<hr /><em>This discussion was created from the release <a href='https://github.com/gaborauth/toolsboox-android/releases/tag/v1.5.15-00'>v1.5.15-00</a>.</em>

**Comment by AxelSteinhage** [comment]
Hi,
is this the right place to tell experience and observations (would not like to call it "bugs" :) ) I have with the app?
Axel
